three.js/src/new-api/arjs-session.js

```diff
@@ -109,11 +105,11 @@ const Session = function(parameters){
 
 				console.log(
 						'actual source dimensions',
-						arSource.domElement.videoWidth,
-						arSource.domElement.videoHeight
+						arSource.domElement.clientWidth,
```

Comment:
`ClientWidth` and `clientHeight` instead are available, we can use them  for now. Now the orientation is correct.